main/src/ca/uwaterloo/flix/language/phase/Jvm.scala

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2024 Jonathan Lindegaard Starup
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ca.uwaterloo.flix.language.phase
+
+import java.lang.reflect.{Field, Member, Method}
+
+object Jvm {
+
+  /**
+    * Returns the (static/dynamic) `fieldName` field of `clazz` if it exists.
+    *
+    * Field name "length" of array classes always return `None` (see Class.getField).
+    *
+    * @param clazz the class to search
+    * @param static whether to find a static field or an instance field
+    */
+  def getField(clazz: Class[?], fieldName: String, static: Boolean): Option[Field] = {
+    try {
+      val field = clazz.getField(fieldName)
+      if (isStatic(field) == static) Some(field) else None
+    } catch {
+      case _: NoSuchFieldException => None
+    }
+  }
+
+  /** Returns `true` if `member` is a static member ([[Field]] and [[Method]] extends [[Member]]). */
```

Comment:
Can we not do normal doc comments?

Reply:
what part of it do you mean? I didn't know `Member` existed so I though the clarification was nice

Reply:
I mean not to use singleline docs. Your new found style which is not used anywhere else :)

Reply:
> which is not used anywhere else :)

The bane of innovation and improvement! ✊

I can change it - but I do think its better